CMakeLists.txt

```diff
@@ -184,21 +184,27 @@ set(Boost_USE_STATIC_LIBS OFF)
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_USE_STATIC_RUNTIME OFF)
 # necessary for Windows and RHEL <=6 systems
-set(Boost_NO_BOOST_CMAKE ON)
+# set(Boost_NO_BOOST_CMAKE ON)
 
 if(WIN32)
-add_definitions( -DHAVE_SNPRINTF) #Python decides to overwrite snprintf if this is not defined, because ofcourse it does.
+  add_definitions( -DBOOST_ALL_NO_LIB )
```

Comment:
Why do we need this? Current Gadgetron compiles and runs under Windows 10 VS 2019